proto/forch_configuration.proto

```diff
@@ -196,6 +199,17 @@ message ProxyServerConfig {
  * Proxy targets
  */
 message ProxyTarget {
-    // port number for target
-    int32 port = 1;
+  // port number for target
+  int32 port = 1;
 }
+
+/*
+ * Monitoring for dataplane
+ */
+message DataplaneMonitoring {
+  // vlan packet rate polling interval in seconds
+  int32 pkt_rate_interval_sec = 1;
```

Comment:
I'd use the more general gauge_metrics_interval_sec here -- since the scanning interval would likely be used for more than just pkt_rate

Reply:
Updated